processing/src/main/java/io/druid/segment/IndexMergerV9.java

```diff
@@ -726,7 +727,8 @@ private void mergeIndexesAndWriteColumns(
   private LongColumnSerializer setupTimeWriter(final IOPeon ioPeon) throws IOException
   {
     LongColumnSerializer timeWriter = LongColumnSerializer.create(
-        ioPeon, "little_end_time", CompressedObjectStrategy.DEFAULT_COMPRESSION_STRATEGY
+        ioPeon, "little_end_time", CompressedObjectStrategy.DEFAULT_COMPRESSION_STRATEGY,
+        CompressionFactory.DEFAULT_LONG_ENCODING
```

Comment:
time uses the default compression strategy and isn't set in indexSpec, so I'm not sure whether to do the same for encoding considering time is a special column